eng/pipelines/templates/jobs/apiview-review-gen-python.yml

```diff
@@ -0,0 +1,48 @@
+pr: none
+
+trigger: none
+
+pool:
+  name: azsdk-pool-mms-ubuntu-2004-general
+  vmImage: MMSUbuntu20.04
+
+parameters:
+  - name: Reviews
+    type: string
+    default: ''  
+  - name: APIViewURL
+    type: string
+    default: '' 
+  - name: StorageContainerUrl
+    type: string
+    default: ''
+  - name: Language
+    type: string
+    default: ''
+
+variables:
+  NugetSecurityAnalysisWarningLevel: none
+  skipComponentGovernanceDetection: true
+  PythonIndexUrl: 'https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-python/pypi/simple/'
+  PythonVersion: '3.10.2'
+  ApiStubVersion: '0.3.1'
+
+jobs:
+- job: CreateReviewCodeFile
+  displayName: Create APIView code file
+  steps:
+  - template: /eng/pipelines/templates/steps/use-python-version.yml
+    parameters:
+      versionSpec: '$(PythonVersion)'
+
+  - script: |
+      pip install virtualenv aiohttp chardet trio
+      pip install api-stub-generator==$(ApiStubVersion) --index-url $(PythonIndexUrl)
+    displayName: 'Install api-stub-generator'
+
+  - template: /eng/pipelines/templates/steps/apiview-review-gen.yml
+    parameters:
+      Reviews: ${{ parameters.Reviews }}
+      APIViewURL: ${{parameters.APIViewURL }}
+      StorageContainerUrl: ${{ parameters.StorageContainerUrl }}
+      Language: ${{ parameters.Language }}
```

Comment:
Let's just pass in the path to the script instead of the language.

Reply:
done